### PR TITLE
Updated Erlang version to 17.4

### LINKED ISSUE #8: Outdated Erlang version causes RabbitMQ to disable SSL listeners
When mounting this custom configuration file on `/etc/rabbitmq/rabbitmq.config`:
```
[
  {rabbit, [
    {loopback_users, []},
    {ssl_listeners, [5671]},
    {ssl_options, [
      {cacertfile,"/etc/rabbitmq/testca/cacert.pem"},
      {certfile,"/etc/rabbitmq/server/cert.pem"},
      {keyfile,"/etc/rabbitmq/server/key.pem"},
      {verify,verify_peer},
      {fail_if_no_peer_cert,false}
    ]},
    {tcp_listeners, [
      {"0.0.0.0", 5672}
    ]}
  ]}
].
```

The rabbitmq container log outputs the following warning:

```
=ERROR REPORT==== 3-Feb-2015::16:41:33 ===
The installed version of Erlang (R15B01) contains the bug OTP-10905,
which makes it impossible to disable SSLv3. This makes the system
vulnerable to the POODLE attack. SSL listeners for AMQP have therefore
been disabled.

You are advised to upgrade to a recent Erlang version; R16B01 is the
first version in which this bug is fixed, but later is usually
better.

If you cannot upgrade now and want to re-enable SSL listeners, you can
set the config item 'ssl_allow_poodle_attack' to 'true' in the
 'rabbit' section of your configuration file.
```

I can get the Dockerfile to install an updated Erlang version.

### PR DESCRIPTION
**Pull request by vovimayhem**
Updated Erlang version to 17.4, installed from the official Erlang debian repo, in case the switch to Debian Jessie can't be done right now.

Fixes #8

### CODE REVIEW

**Comment by tianon** [comment]
Calling it "official" is a bit of a stretch: it's officially endorsed, but it's not run by the Erlang project directly (although it is a neat solution to our issue).

http://www.erlang.org/download.html
> Erlang/OTP is also distributed in pre built packages courtesy of Erlang Solutions.

File: Dockerfile [hunk 1]
```diff
@@ -3,6 +3,9 @@ FROM debian:wheezy
 # add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
 RUN groupadd -r rabbitmq && useradd -r -d /var/lib/rabbitmq -m -g rabbitmq rabbitmq
 
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv D208507CA14F4FCA
```
Comment (yosifkit) [inline Dockerfile]: Can we get the full fingerprint here and use `pool.sks-keyservers.net` for better reliability?

**Comment by vovimayhem** [comment]
Let me see...

**Comment by vovimayhem** [comment]
I can add the keys of both Erlang & RabbitMQ in the same apt-key adv command. Should I?

**Comment by tianon** [comment]
Naw, this is good IMO. :+1:

It might be worth adding a link to both the Erlang downloads page that mentions that these packages are endorsed and the downloads page itself, though.

**Comment by tianon** [comment]
Hey!  Sorry, GitHub doesn't tell us when the PR is updated, so we had no idea this was ready for re-review!

That comment looks perfect, but I'm puzzled by the unrelated whitespace changes further down?

File: Dockerfile [hunk 2]
```diff
@@ -15,8 +22,8 @@ RUN apt-get update && apt-get install -y rabbitmq-server=$RABBITMQ_VERSION --no-
 # get logs to stdout (thanks to http://www.superpumpup.com/docker-rabbitmq-stdout for inspiration)
 # TODO figure out what we'd need to do to add "(sasl_)?" to this sed and have it work ("{"init terminating in do_boot",{rabbit,failure_during_boot,{error,{cannot_log_to_tty,sasl_report_tty_h,not_installed}}}}")
 RUN sed -E 's!^(\s*-rabbit\s+error_logger)\s+\S*!\1 tty!' /usr/lib/rabbitmq/lib/rabbitmq_server-*/sbin/rabbitmq-server > /tmp/rabbitmq-server \
-	&& chmod +x /tmp/rabbitmq-server \
-	&& mv /tmp/rabbitmq-server /usr/lib/rabbitmq/lib/rabbitmq_server-*/sbin/rabbitmq-server
+  && chmod +x /tmp/rabbitmq-server \
+  && mv /tmp/rabbitmq-server /usr/lib/rabbitmq/lib/rabbitmq_server-*/sbin/rabbitmq-server
```
Comment (yosifkit) [inline Dockerfile]: Can we drop these unrelated whitespace changes?

**Comment by vovimayhem** [comment]
Right away! Sorry for that!

**Comment by vovimayhem** [comment]
Ready :)

**Comment by tianon** [comment]
LGTM!

**Comment by yosifkit** [comment]
LGTM
